### DISCUSSION TITLE
Can't use (<= , <) operators on filter DateTime field

### DISCUSSION THREAD

**Discussion by oscar-costa** [pocketbase/pocketbase#1598] [Q&A] [answered]
Hi.

I have a collection with a dateTime field and only one record. For some reason I can't manage to use the "=" and "<=" operators when filtering a search using the JS SDK.  ">=" operator does work has expected

The field is called date_start and, in the database, has the value of: 2023-01-09 00:00:00

This is my fetch code:
`
const data = await usePocketBase.collection('calendar').getList(
      1,
      20,
      {
        filter: 'date_start = "2023-01-09 00:00:00"',
        expand: 'user_id'
      }
    )
`
this code returns no values, but if i change the filter value to 'date_start >= "2023-01-09 00:00:00"', I do get one record has expected.

**Comment by ganigeorgiev** [comment, marked as answer]
The full datetime format is "Y-m-d H:i:s.uZ" (eg. "2023-01-14 16:35:15.123Z").

When using the exact compare operators (aka. `=`/`!=`) you have to provide the complete datetime string. You can also provide a range filter if you want to target an entire day/period, something like:

```js
date_start >= "2023-01-09 00:00:00.000Z" && date_start <= "2023-01-09 23:59:59.999Z"
```

Alternatively, you can also use the `~` operator, eg.:
```js
date_start ~ "2023-01-09"
```

**Comment by oscar-costa** [comment]
Thanks, it worked.
